README.md

```diff
@@ -1,3 +1,3 @@
 # CACAO2021
 
-A simulation of the cacao market for the CODEVSI course.
```

Comment:
You can replace this with the version from the main repository. To do that, right click on this file -> Replace with -> Branch, Tag or Reference -> Remote tracking -> select the main repository -> Replace. Then commit and push to your repository. Don't close this pull request, it will be automatically updated.

Reply:
Actually I tried to do it but now it seems that even if the .md document is the same, it is still seeing it as if we were changing it.

Reply:
Did you try to replace it with origin/master?

Reply:
When I'm trying to replace the file in Eclipse it doesn't do anything because the file is alreay the same as in the origin/master folder

Reply:
The problem might be fixed now.